src/ci.yml

```diff
@@ -14,7 +14,9 @@ pr:
   paths:
     include:
     - src/
-
+    - test/DotNetWorkerTests
```

Comment:
We may want to be broader than this even. Changes to props or targets, or other test files could cause CI issues which won't be discovered until a later PR.

Reply:
SDK has been added by a separate change, so that is now included.